schematic/utils/df_utils.py

```diff
@@ -5,12 +5,12 @@
 import logging
 from copy import deepcopy
 from time import perf_counter
-from typing import Union
 from datetime import datetime
+from typing import Union, Optional
 import dateparser as dp
 import pandas as pd
 import numpy as np
-from pandarallel import pandarallel  # type: ignore
```

Comment:
Removing this will cause a mypy error

Reply:
I think this was removed by something automatically. I'll add it back. thanks for catching